magenta-music/.pylintrc

```diff
@@ -0,0 +1,219 @@
+[MASTER]
```

Comment:
I think you can leave this file out and it will default to the other one.

Reply:
I guess, unless pytest copies the package to some temporary location.

Let me know if any action is required from me.